osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -138,53 +142,71 @@ private void updateStateTransforms(DrawableHitObject drawableHitObject, ArmedSta
                 switch (state)
                 {
                     case ArmedState.Hit:
+
+                        bool useFlash = config.Get<bool>(OsuSetting.HitLighting);
```

Comment:
Rather than doing a `config.Get` each time, it's better to retrieve the bindable in a `BackgroundDependencyLoader` method, even if not handling the value changed event. This is due to lower overheads. See the following example (there are hundreds more):

https://github.com/ppy/osu/blob/7bc8908ca9c026fed1d831eb6e58df7624a8d614/osu.Game/Beatmaps/Drawables/Cards/Buttons/DownloadButton.cs#L40-L44

Reply:
True, I was focused about getting the animation right that I forgot about that